src/SessionManager/WebBrowser.h

```diff
@@ -0,0 +1,39 @@
+/* This file is part of the CARTA Image Viewer: https://github.com/CARTAvis/carta-backend
+   Copyright 2018, 2019, 2020, 2021 Academia Sinica Institute of Astronomy and Astrophysics (ASIAA),
+   Associated Universities, Inc. (AUI) and the Inter-University Institute for Data Intensive Astronomy (IDIA)
+   SPDX-License-Identifier: GPL-3.0-or-later
+*/
+
+#ifndef CARTA_BACKEND_SRC_SESSIONMANAGER_WEBBROWSER_H_
+#define CARTA_BACKEND_SRC_SESSIONMANAGER_WEBBROWSER_H_
+
+#include <string>
+#include <vector>
+
+namespace carta {
+class WebBrowser {
+public:
+    WebBrowser(const std::string&, const std::string&);
+    ~WebBrowser() = default;
+    bool Status() const {
+        return _status;
+    };
+    std::string Error() const {
+        return _error;
+    }
+
+private:
+    void ParseCmd();
+    void OpenSystemBrowser();
+    void OpenBrowser();
+    bool _status;
+    bool _path_exists;
```

Comment:
I can only see `_path_exists = true`. Where are you initialising this?

Reply:
good catch, thanks